src/Exception/MultipartUploadException.php

```diff
@@ -1,12 +1,15 @@
 <?php
 namespace Aws\Exception;
 
+use Aws\CommandInterface;
 use Aws\Multipart\UploadState;
 
 class MultipartUploadException extends \RuntimeException
 {
     /** @var UploadState State of the erroneous transfer */
     private $state;
+    /** @var string File path of transfer object */
+    private $filePath;
 
     /**
      * @param UploadState      $state Upload state at time of the exception.
```

Comment:
Side note: It's really weird that `$prev` might be an array. We should look at this and see if we can make $prev always an exception in a backwards compatible way so it's a bit more predictable.